pkg/options/options.go

```diff
@@ -73,6 +73,17 @@ type XcodeBuild struct {
 	Configuration      string      `help:"The configuration used to build the application"`
 }
 
+type XcodeArchive struct {
+	Path               utils.Paths `arg:"" name:"path" help:"The path to the directory or file to upload" type:"path" default:"."`
+	IgnoreEmptyDsym    bool        `help:"Throw warnings instead of errors when a dSYM file is found, rather than the expected dSYM directory"`
+	IgnoreMissingDwarf bool        `help:"Throw warnings instead of errors when a dSYM with missing DWARF data is found"`
+	Scheme             string      `help:"The name of the Xcode options.Scheme used to build the application"`
+	VersionName        string      `help:"The version of the application"`
+	XcodeProject       utils.Path  `help:"The path to an Xcode project, workspace or containing directory from which to obtain build information" type:"path"`
+	ProjectRoot        string      `help:"The path to strip from the beginning of source file names referenced in stacktraces on the BugSnag dashboard" type:"path"`
+	Configuration      string      `help:"The configuration used to build the application"`
+}
```

Comment:
This looks identical to xcode-build, I don't think we use them all do we? e.g. XcodeProject

It would be good if they could inherit a common superset though perhaps?

The Path text could do with more detail on what you can pass in